win32/devdoc/threadpool_win32.md

```diff
@@ -293,12 +293,12 @@ MOCKABLE_FUNCTION(, int, threadpool_schedule_work_item, THANDLE(THREADPOOL), thr
 ### threadpool_destroy_work_item
 
 ```c
-MOCKABLE_FUNCTION(, void, threadpool_destroy_work_item, THANDLE(THREADPOOL), threadpool, THREADPOOL_WORK_ITEM_HANDLE, work_item_context);
+static void threadpool_destroy_work_item(THANDLE(THREADPOOL_WORK_ITEM) work_item_context);
```

Comment:
>work_item_context [](http://example.com/codeflow?start=71&length=17)

Let's name this what it is: threadpool_work_item

Reply:
Of course in the specs below also.

Reply:
Done